Jenkinsfiles/maint-mode.yml

```diff
@@ -0,0 +1,3 @@
+pipeline:
```

Comment:
@safwanrahman It is unclear why you object to this file. It seems fine to me, and I appreciate the live server.